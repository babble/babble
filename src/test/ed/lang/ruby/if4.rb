
def blah(z)
  gar = nil;
  foo = 
    if z == 5
      gar = 1
      "five"
    elsif z == 6
      gar = 2
      "six"
    elsif z == 7
      gar = 3
      "six"
    else
      gar = 4
      "stupid"
    end
  puts gar
  return foo
end

puts blah(5)
puts blah(6)
puts blah(7)
puts blah(8)
