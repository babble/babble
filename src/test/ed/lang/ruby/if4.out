1
five
2
six
3
six
4
stupid
